tests/wpt/mozilla/meta/MANIFEST.json

```diff
@@ -25940,6 +25945,10 @@
    "333a80e3d4aec4a8be5566605fae5675a821085b",
    "support"
   ],
+  "mozilla/referrer-policy/generic/subresource/mozresource.pyc": [
+   "c3d7573593042e1d218e532071225d92c4920954",
+   "support"
+  ],
```

Comment:
You'll need to remove the pyc files from your working tree and run manifest-update again.